docs/LogGuide.md

```diff
@@ -760,26 +763,33 @@ to include both possibilities.
 
 Ground AOEs that don't hit anybody are considered [NetworkAOEAbility](#line22) lines.
 
+There are two fields on 21/22 lines that provide information about the number of targets affected. The `targetCount`
+field indicates the number of targets. The `targetIndex` field indicates which target this particular line refers to.
+For example, for a 21-line, you would see a `targetIndex` of 0, and a `targetCount` of 1. For an AoE ability that hits
+three targets, all three lines would have a `targetCount` of 3, but the `targetIndex` would be 0, 1, and 2 for the three
+lines. Thus, you can determine all of the targets hit by a particular ability usage by collecting 21/22-lines until you
+see one for which `targetCount - 1 == targetIndex`.
+
 <!-- AUTO-GENERATED-CONTENT:START (logLines:type=Ability&lang=en-US) -->
 
 #### Structure
 
 ```log
 Network Log Line Structure:
-21|[timestamp]|[sourceId]|[source]|[id]|[ability]|[targetId]|[target]|[flags]|[damage]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[?]|[targetCurrentHp]|[targetMaxHp]|[targetCurrentMp]|[targetMaxMp]|[?]|[?]|[targetX]|[targetY]|[targetZ]|[targetHeading]|[currentHp]|[maxHp]|[currentMp]|[maxMp]|[?]|[?]|[x]|[y]|[z]|[heading]|[sequence]|[targetIndex]
+21|[timestamp]|[sourceId]|[source]|[id]|[ability]|[targetId]|[target]|[flags1]|[damage1]|[flags2]|[damage2]|...|[flags8]|[damage8]|[targetCurrentHp]|[targetMaxHp]|[targetCurrentMp]|[targetMaxMp]|[?]|[?]|[targetX]|[targetY]|[targetZ]|[targetHeading]|[currentHp]|[maxHp]|[currentMp]|[maxMp]|[?]|[?]|[x]|[y]|[z]|[heading]|[sequence]|[targetIndex]|[targetCount]
```

Comment:
Did you edit this manually? If so, please edit `resources/netlog_defs.ts` and then rerun `util/gen_log_guide.ts`.

I think also you shouldn't rename the first `flags` as there is code (possibly user code) using those field names.

Alternatively you could just leave it as-is and only add `targetCount`.

Reply:
It is all manual, I do not have a local environment for any of this stuff. I am not intending to make any code changes, just purely docs since the information isn't specific to cactbot.

Reply:
If you aren't able to run that script, then can you undo all of your changes to autogenerated code blocks? (see: `<!-- AUTO-GENERATED-CONTENT:START (logLines:type=Ability&lang=en-US) -->`)

Reply:
Maybe the guide should just be split between the "general log guide" (since everyone, including me, links this whenever someone asks about log lines) and cactbot-specific stuff? I agree that someone referencing it for cactbot specifically would want to see information that is relevant to cactbot, but it seems most people just reference it for general log lines, for which it would be outdated. It's the closest thing there is to general documentation about log lines.

Reply:
Sorry, it's just that it doesn't make sense to make manual updates to an autogenerated section.  The next time anybody runs that script it will clobber all of your changes.